website/docs/docs/deploy/continuous-integration.md

```diff
@@ -32,7 +32,6 @@ The [dbt Cloud scheduler](/docs/deploy/job-scheduler) executes CI jobs different
 
 - **Concurrent CI checks** &mdash; CI runs triggered by the same dbt Cloud CI job execute concurrently (in parallel), when appropriate
 - **Smart cancellation of stale builds** &mdash; Automatically cancels stale, in-flight CI runs when there are new commits to the PR
-- **Run slot treatment** &mdash; CI runs don't consume a run slot
```

Comment:
i think this is stale info. removed.

Reply:
CI runs don't consume run slots for free, team, & enterprise accounts

Reply:
we removed the whole section pertaining to "run slots" in pr #4166 .  the bullet should've been removed from the list too since there's no corresponding H3 anymore. catching that now.

if we want to call this out again, should i go ahead and add the section back with the extra "free, team, enterprise" callout?